ironfish/src/network/peers/connections/webSocketConnection.ts

```diff
@@ -86,12 +86,15 @@ export class WebSocketConnection extends Connection {
         return
       }
 
+      this.metrics?.p2p_InboundTraffic.add(event.data.byteLength)
```

Comment:
moving these up here so that we record inbound traffic even if the message fails to parse. We were doing this for WebRTC already